src/utils/getPreparedPeople.ts

```diff
@@ -0,0 +1,14 @@
+import { Person } from '../types';
+
+export const getPreparedPeople = (people: Person[]) => {
+  let preparedPeople = [...people];
+
+  preparedPeople = preparedPeople.map(person => {
```

Comment:
```suggestion
  return preparedPeople.map(person => {
```
`map` returns copy of array, so you could do this